tensorflow/training/docker/2.6/py3/cu112/Dockerfile.gpu

```diff
@@ -0,0 +1,330 @@
+FROM nvidia/cuda:11.2.1-base-ubuntu20.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# prevent stopping by user interaction
+ENV DEBIAN_FRONTEND noninteractive
+ENV DEBCONF_NONINTERACTIVE_SEEN true
+ENV SAGEMAKER_TRAINING_MODULE sagemaker_tensorflow_container.training:main
+
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONIOENCODING=UTF-8
+ENV LANG=C.UTF-8
+ENV LC_ALL=C.UTF-8
+
+# Set environment variables for MKL
+# For more about MKL with TensorFlow see:
+# https://www.tensorflow.org/performance/performance_guide#tensorflow_with_intel%C2%AE_mkl_dnn
+ENV KMP_AFFINITY=granularity=fine,compact,1,0
+ENV KMP_BLOCKTIME=1
+ENV KMP_SETTINGS=0
+ENV RDMAV_FORK_SAFE=1
+
+ARG PYTHON=python3.8
+ARG PYTHON_PIP=python3-pip
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.8.2
```

Comment:
Not a requirement right now.

Can we upgrade to https://www.python.org/downloads/release/python-3812/ on the next release?